cms_helpers/filer_fields.py

```diff
@@ -47,7 +48,10 @@ def __init__(self, verbose_name=None, *args, **kwargs):
         self.extensions = kwargs.pop('extensions', None)
         self.alt_text_required = kwargs.pop('alt_text_required', True)
 
-        super().__init__(*args, **kwargs)
```

Comment:
I would do a `kwargs.setdefault('on_delete', models.CASCADE)` instead of "trying" to super. This might have weird side effects.